Add Analyze index disk usage API example (#134282)
elastic/elasticsearch: docs/reference/elasticsearch/rest-apis/index-disk-usage.md

                "inverted_index": {
                    "total": "22mb",
                    "total_in_bytes": 23090908
                },
                "stored_fields": "0b",
                "stored_fields_in_bytes": 0,
                "doc_values": "0b",
                "doc_values_in_bytes": 0,
                "points": "0b",
                "points_in_bytes": 0,
                "norms": "2.3kb",
                "norms_in_bytes": 2356,
                "term_vectors": "2.2kb",
                "term_vectors_in_bytes": 2310,
                "knn_vectors": "0b",
                "knn_vectors_in_bytes": 0
            },
            "context.keyword": {...},
            "message": {...},
            "message.keyword": {...}
        }
    }
}
```

% TESTRESPONSE[s/: \{\.\.\.\}/: $body.$_path/]
% TESTRESPONSE[s/: (\-)?[0-9]+/: $body.$_path/]
% TESTRESPONSE[s/: "[^"]*"/: $body.$_path/]

1. The store size of only analyzed shards of the index.

2. The total size of fields of the analyzed shards of the index. This total
is usually smaller than the index size specified in `<1>` as some small metadata
files are ignored and some parts of data files might not be scanned by the API.

> 3. The stored size of the `_id` field

Reviewer: ```suggestion
3. The disk space used by `_id` field values for direct document retrieval. This storage enables [getting a document by its ID](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-get), without needing to search the inverted index.
```

might be helpful context

> 4. The stored size of the `_source` field. As stored fields are stored

Reviewer: ```suggestion
4. The disk space used by the `_source` field. As stored fields are stored
```
